galaxy_ng/app/api/ui/views/landing_page.py

```diff
@@ -0,0 +1,161 @@
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.response import Response
+from pulp_ansible.app.models import (
+    CollectionVersion,
+    AnsibleDistribution
+)
+from galaxy_ng.app.models import Namespace
+from django.conf import settings
+from galaxy_ng.app.api import base as api_base
+
+golden_name = settings.GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH
+
+
+class LandingPageView(api_base.APIView):
+    permission_classes = [IsAuthenticatedOrReadOnly]
+
+    def get(self, request, *args, **kwargs):
+        distro = AnsibleDistribution.objects.get(base_path=golden_name)
+        repository_version = distro.repository.latest_version()
+        versions = CollectionVersion.objects.filter(pk__in=repository_version.content)
+        collection_count = versions.select_related('collection').count()
+        partner_count = Namespace.objects.count()
+
+        data = {
+            'estate': {
+                'items': [
+                    {
+                        'id': 'ansible-collections',
+                        'count': collection_count,
+                        'shape': {
+                            'title': 'Collections',
+                            'href': './ansible/automation-hub/',
+                        },
+                        'permissions': [{
+                            'method': 'isEntitled',
+                            'args': ['ansible'],
+                        }],
+                    },
+                    {
+                        'id': 'ansible-partners',
+                        'count': partner_count,
+                        'shape': {
+                            'title': 'Partners',
+                            'href': './ansible/automation-hub/partners/',
+                        },
+                        'permissions': [{
+                            'method': 'isEntitled',
+                            'args': ['ansible'],
+                        }],
+                    },
+                ],
+            },
+            'recommendations': {},  # Hub currently does not have recommendation items.
```

Comment:
The daynum has the chance not showing up some of the namespaces, so to avoid discrimination I used a random generator to do this (if we mind that on every reload there could be a different one I can go with the daynum as you wrote above). I also did my best to do the title and description it, but I probably need to talk to somebody what should it actually say.

Reply:
Taufique and Tina approved the current looks (icon + title + description)